docs/roles/project-coordinator.rst

```diff
@@ -8,32 +8,43 @@ project lead for a specific project. This page explains more about the
 responsibilities and role of the project coordinator.
 
 
-Description
+Description 
 -----------
 
-The project coordinator is responsible for ensuring that all club projects have the resources needed to continue developing. This may include monetary resources, connections to mentors or the like and infrastructure to host the projects. 
+The project coordinator is responsible for ensuring that all club projects have
+the resources needed to continue developing.  This may include monetary
+resources, connections to mentors or the like and infrastructure to host the
```

Comment:
Change sentence structure. "...monetary resources, connections to mentors or the like, and infrastrcture to host..."

Reply:
@axk4545 @jflory7 You guys wrote this, I'm going to defer to you.

Reply:
@Serubin only difference is a comma for clarity as far as I can tell. Also, fix the random newline

Reply:
@axk4545 @jflory7 Ping ping - how would you guys liked this changed?

Reply:
@Tjzabel What would you suggest in place here?